react-native/react/native/listen-log-ui.js

```diff
@@ -3,8 +3,9 @@ import {logUiLog} from '../actions/notifications'
 
 export default function ListenLogUi () {
   engine.listenOnConnect('ListenLogUi', () => {
-    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}) => {
+    engine.listenGeneralIncomingRpc('keybase.1.logUi.log', (params: {text: Text, level: LogLevel}, response:any) => {
```

Comment:
It used to, but now it waits after firing all of them for a message in order to preserve ordering.